### PR TITLE
Task/optimize network api

### PR DESCRIPTION
**Pull request by duongvonga2**

### CODE REVIEW

File: packages/dashboard-api/src/modules/networks/model.js [hunk 1]
```diff
@@ -91,9 +106,61 @@ async function updateFiatVolume(
...
 }
 
+async function updateFiatVolumeV2(
+  networks = [],
+  tokensVolume24h = [],
+  tokensVolumeAllTime = [],
+  totalMintTokens = [],
+  totalBurnTokens = [],
+) {
+  let tokenNames = tokensVolume24h.concat(tokensVolumeAllTime).concat(totalMintTokens).concat(totalBurnTokens).map(item => item.tokenName);
```
Comment (sanglt1902) [inline packages/dashboard-api/src/modules/networks/model.js:116]: You can use `spread syntax` to simplify

File: packages/dashboard-api/src/modules/networks/model.js [hunk 2]
```diff
@@ -91,9 +106,61 @@ async function updateFiatVolume(
...
+  tokenNames = [...new Set(tokenNames)];
+  const tokenPrices = await getPricesByTokenNames(tokenNames);
+  const tokenPriceProperties = setKeyValueTokenPrices(tokenPrices);
```
Comment (sanglt1902) [inline packages/dashboard-api/src/modules/networks/model.js:119]: You can use prop/propty for short form of properties

File: packages/dashboard-api/src/modules/networks/model.js [hunk 3]
```diff
@@ -91,9 +106,61 @@ async function updateFiatVolume(
...
+    networkInfo.usdAllTime = numberToFixedAmount(USDAllTime);
+    networkInfo.mintFee = USDMint;
+    networkInfo.burnFee = USDBurn;
+  }
+  return networks;
+}
 async function getNetworkById(networkId) {
   //	Get token_name from registered_tokens table related to networkId
-  const tokens = (await getTokensbyNetworkId(networkId)).map(element => element.token_name);
+  const tokens = (await getTokensbyNetworkId(networkId)).map((element) => element.token_name);
```
Comment (sanglt1902) [inline packages/dashboard-api/src/modules/networks/model.js:163]: Add `()` or not is the same

**Review by sanglt1902 — APPROVED** [review]
Please take a look at my comments. The rest LGTM

File: packages/dashboard-api/src/modules/networks/model.js (hunk 3 above)
Comment (duongvonga2) [reply packages/dashboard-api/src/modules/networks/model.js:163]: it is default config when i use format shortcut

**Review by sanglt1902 — APPROVED** [review]

**Pull request merged**